compaction/compaction_manager.cc

```diff
@@ -755,7 +755,9 @@ future<> compaction_manager::rewrite_sstables(column_family* cf, sstables::compa
 }
 
 future<> compaction_manager::perform_sstable_scrub_validate_mode(column_family* cf) {
-    return run_custom_job(cf, sstables::compaction_type::Scrub, [this, &cf = *cf, sstables = get_candidates(*cf)] () mutable -> future<> {
+    // All sstables must be included, even the ones being compacted, such that everything in table is validated.
```

Comment:
Note that without a full barrier we can still miss sstables that are sealed after validate starts.

Reply:
true, but I don't think it's a real problem.

Reply:
yes, if the inputs are valid, the output should be too